tfhub_dev/assets/sayakpaul/deeplabv3-xception65-ade20k/1.md

```diff
@@ -0,0 +1,9 @@
+# Placeholder sayakpaul/deeplabv3-xception65-ade20k/1
```

Comment:
Instead of creating a new Placeholder MD and Lite MD, is it possible to simply move:

sayakpaul/deeplabv3-mobilenetv2-xception65/1.md (instead of this new file)
and
sayakpaul/deeplabv3-mobilenetv2-xception65/default/lite/1.md (instead of the sayakpaul/deeplabv3-xception65-ade20k/default/lite/1.md) ?

Reply:
Sorry for the confusion, I'm not very familiar with GitHub workflows. 

My idea was that the files added in this PR (`tfhub_dev/assets/sayakpaul/deeplabv3-xception65-ade20k/1.md` and `tfhub_dev/assets/sayakpaul/deeplabv3-xception65-ade20k/default/lite/1.md`) have the same content as the already existing `sayakpaul/deeplabv3-xception65-ade20k/default/lite/1.md` and `sayakpaul/deeplabv3-xception65-ade20k/default/lite/1.md`. As you say the latter directory name was a bit off, hence the change.

Creating these new files from scratch would keep the existing files and duplicate the content. So I suggested to instead do a `git move` (or something equivalent) from the `deeplabv3-mobilenetv2-xception65` directory to the new `deeplabv3-xception65-ade20k` directory.

I might be missing something. Let me know if this doesn't make sense.

Reply:
So instead of renaming, you are suggesting to do `git mv`?

Reply:
I think the important part is to not keep the same content in two markdown files, so we if we add `tfhub_dev/assets/sayakpaul/deeplabv3-xception65-ade20k/1.md` then we should delete `tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2-xception65/1.md`. 

I think git mv + add the remaining changes to the moved file would achieve that, but any other method also sounds good to me.

Reply:
I don't see `tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2-xception65/1.md` anyway and the contents of `tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2-xception65/1.md` and `tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2-xception65/2.md` are different if you look closely. If I am missing out on something please let me know.

Reply:
Sorry for the back-and-forth, I am probably missing something.

This is the link to `tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2-xception65/1.md`:

https://raw.githubusercontent.com/tensorflow/hub/master/tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2-xception65/1.md

which is identical to the file added in this PR `tfhub_dev/assets/sayakpaul/deeplabv3-xception65-ade20k/1.md`:

https://raw.githubusercontent.com/tensorflow/hub/c30a2f5a80878c68ef455c7f364e27560a4eb039/tfhub_dev/assets/sayakpaul/deeplabv3-xception65-ade20k/1.md

The problem is that this PR does not delete `tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2-xception65/1.md`. The two files have the same model handle (`sayakpaul/deeplabv3-xception65-ade20k/1`) which is meant to be a unique identifier, so merging this PR would result in an invariant violation.

Reply:
Should be fixed now.

Reply:
Thanks a lot @sayakpaul , this PR looks good now. Our automation should pick this up in the next hours. I'll let you know once these changes are live.

Reply:
Thanks a lot for your guidance @maringeo